frontend/src/components/core/Block/Block.tsx

```diff
@@ -167,22 +143,29 @@ class Block extends PureComponent<Props> {
   ): ReactNode | null {
     const element = reportElement.get("element")
 
-    if (element.get("type") === "empty") {
-      // Just a plain div -- so we're sure the rendered element has no
-      // height/margin/padding. Also saves some CPU cycles as a side-effect.
-      return <div className="stEmpty" key={index}></div>
-    }
-
     const component = this.renderElement(
       element,
       index,
       width,
       reportElement.get("metadata")
     )
 
+    const componentWithMaybe = (
+      <Maybe
+        enable={
+          component !== undefined ||
+          this.props.reportRunState !== ReportRunState.RUNNING
+        }
+      >
+        {" "}
```

Comment:
Which is the purpose of this white spaces?

Reply:
I should investigate this a bit more. Honestly, I thought they were required.

Reply:
I think I found the problem here. Will fix.

Reply:
Seems to be related to this: https://stackoverflow.com/questions/45789244/react-component-errors-undefined-but-should-return-null

Reply:
I took the `undefined` out. So now, we never return `undefined` from `renderElement` (similar tot the way it was)